queue_services/entity-filer/requirements.txt

```diff
@@ -24,6 +23,7 @@ urllib3==1.26.11
 minio==7.0.2
 PyPDF2==1.26.0
 reportlab==3.6.12
+git+https://github.com/bcgov/sbc-connect-common.git#egg=gcp-queue&subdirectory=python/gcp-queue
 git+https://github.com/bcgov/business-schemas.git@2.18.27#egg=registry_schemas
-git+https://github.com/bcgov/lear.git#egg=legal_api&subdirectory=legal-api
 git+https://github.com/bcgov/lear.git#egg=entity_queue_common&subdirectory=queue_services/common
+git+https://github.com/thorwolpert/lear.git@requirements-split-nats-2#egg=legal_api&subdirectory=legal-api
```

Comment:
can point back to using bcgov LEAR repo after https://github.com/bcgov/lear/pull/2744 is merged.